usr/src/uts/i86pc/os/intr.c

```diff
@@ -471,6 +472,7 @@
 #include <sys/hypervisor.h>
 #endif
 
+boolean_t kernel_preempt_enable = B_FALSE;
```

Comment:
This seems like the sort of thing that it'd be good to get Bryan to take a look at, but he's on vacation for a fortnight or so.  I'll ping him, in case he's looking for something to do.